src/ocamlorg_web/lib/handler.ml

```diff
@@ -179,6 +179,9 @@ let blog req =
     (Ocamlorg_frontend.blog ~featured ~rss:current_items ~rss_page:page
        ~rss_pages_number:number_of_pages ~news)
 
+let atom =
+  Dream.from_filesystem "data/rss/" "atom.xml"
+
```

Comment:
We can't depend on the filesystem in production, instead we [crunch](https://github.com/mirage/ocaml-crunch) the static file into the server binary.
```suggestion
```